tool/state-processor/src/processor.rs

```diff
@@ -208,7 +213,17 @@ impl<R> State<R> {
 
 		self.mutate_value(p, i, &blake2_128_concat_to_string(h), |a: &mut AccountInfo| {
 			a.data.free += amount;
-			a.data.reserved -= amount;
+
+			if let Some(amount) = a.data.reserved.checked_sub(amount) {
```

Comment:
Please revert this change in this pr, this will break the current state processor unit tests.

Let's do it together with #240